README.md

```diff
@@ -20,39 +20,40 @@ For details of the other Kata Containers repositories, see the
 
 ## Getting Started
 
-* [Installation guides](./install/README.md): Install and run Kata Containers with Docker or Kubernetes
+* [Installation guides](./install/README.md): 
+Install and run Kata Containers with Docker or Kubernetes
 
 ## More User Guides
 
-* [Upgrading](Upgrading.md): how to upgrade from [Clear Containers](https://github.com/clearcontainers) and [runV](https://github.com/hyperhq/runv) to [Kata Containers](https://github.com/kata-containers) and how to upgrade an existing Kata Containers system to the latest version.
-* [Limitations](Limitations.md): differences and limitations compared with the default [Docker](https://www.docker.com/) runtime,
+* [Upgrading](Upgrading.md): How to upgrade from [Intel Clear Containers](https://github.com/clearcontainers) and [runV](https://github.com/hyperhq/runv) to [Kata Containers](https://github.com/kata-containers) and how to upgrade an existing Kata Containers system to the latest version.
```

Comment:
nb: we tend to keep our markdown lines trimmed to 80 chars or so - would you say @jodh-intel ?
I see it as a swings and roundabouts - it makes editing and reflowing lines sometimes more painful, and it has both pros and cons for reviewing patches of markdown (sometimes a win, sometimes a loss).

Reply:
In my experience working on Clear Linux documentation, we set a standard 78 char limit, here: https://docs.01.org/clearlinux/latest/collaboration/structure-formatting.html#white-space-and-line-length.  80 chars seems fine too; I'm not commenting on that.  

Over time, this has proven more useful than not. Of course not all contributors follow this, but  our internal team does. Using this standard helps our team avoid common errors, like code-block syntax breaking due to improper line breaks, or Sphinx directives that are split over line breaks, which also fail. There's also a lot of research that shows humans are more likely to read, or read more, if the char limit is set to around 80.